components/restservice/scripts/creation_validation.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+from os.path import join, dirname
+
+from cloudify import ctx
+
+ctx.download_resource(
+    join('components', 'utils.py'),
+    join(dirname(__file__), 'utils.py'))
+import utils  # NOQA
+
+
+REST_SERVICE_NAME = 'restservice'
+IMMUTABLE_PROPERTIES = [
+    'insecure_endpoints_disabled'
```

Comment:
This really needs #175 to be merged first: then we'll check things like `verify_certificate` and agents credentials too: https://github.com/cloudify-cosmo/cloudify-manager-blueprints/pull/175/files#diff-efe92cfd0125c444af1e24a02f0454faR212